### PR TITLE
document the error event

### PR DESCRIPTION
**Pull request by thijstriemstra**
refs #1810

### CODE REVIEW

File: src/js/player.js [hunk 1]
```diff
@@ -588,6 +588,12 @@ vjs.Player.prototype.onFullscreenChange = function() {
   }
 };
 
+/**
+ * Fired when an error occurred
```
Comment (mmcc) [inline src/js/player.js]: This is probably a nitpick, but to be consistent with the other comments, let's go with "Fired when an error occurs"

**Comment by mmcc** [comment]
:thumbsup: Aside from the minor comment, this looks good to me.

File: src/js/player.js (hunk 1 above)
Comment (thijstriemstra) [inline src/js/player.js]: Pushed a fix.

**Comment by mmcc** [comment]
lgtm!

**Comment by heff** [comment]
looks good!

**Pull request merged**